syncstorage-mysql/src/diesel_ext.rs

```diff
@@ -1,36 +1,70 @@
+use std::{fmt::Debug, marker::PhantomData};
```

Comment:
Not blocking or necessary, but given our goal to make Sync more comprehensible and documented, perhaps a bit of docs in this crate to explain it's to construct the custom wrapper struct for `ON DUPLICATE KEY UPDATE` ORM code for Diesel. To the uninitiated it might be a bit vague.

Reply:
https://mozilla-hub.atlassian.net/browse/STOR-348